pkg/sql/lease.go

```diff
@@ -956,9 +957,10 @@ func makeTableNameCacheKey(dbID sqlbase.ID, tableName string) tableNameCacheKey
 // LeaseManager.mu > tableState.mu > tableNameCache.mu > LeaseState.mu
 type LeaseManager struct {
 	LeaseStore
-	mu       syncutil.Mutex
-	draining bool
-	tables   map[sqlbase.ID]*tableState
+	mu     syncutil.Mutex
```

Comment:
while you're here, can you make this an anonymous struct so it's clear what `mu` is protecting?